housekeeper/store/filters/archive_filters.py

```diff
@@ -0,0 +1,53 @@
+from enum import Enum
+from typing import Callable, List, Optional
+
+from sqlalchemy import or_, and_
+from sqlalchemy.orm import Query
+from sqlalchemy.sql.elements import BooleanClauseList
+
+from housekeeper.store.models import Archive
+
+ARCHIVING_NOT_FINISHED: BooleanClauseList = and_(
+    Archive.archiving_task_id, Archive.archived_at == None
+)
+RETRIEVAL_NOT_FINISHED: BooleanClauseList = and_(
+    Archive.retrieval_task_id, Archive.retrieved_at == None
+)
+
+
+def filter_archiving_in_progress(archives: Query, **kwargs) -> Query:
+    """Return archives where the archiving is not marked as completed."""
+    return archives.filter(ARCHIVING_NOT_FINISHED)
```

Comment:
```suggestion
def filter_archiving_not_finished(archives: Query, **kwargs) -> Query:
    """Return archives where the archiving is not marked as completed."""
    return archives.filter(ARCHIVING_NOT_FINISHED)
```

I see now where the naming from the function above comes from.
Maybe the filters and the function definitions can be streamlined. 

as it is now it jumps between ```not_finished``` and ```in_progress```